atlassian_jwt_auth/contrib/django/decorators.py

```diff
@@ -44,6 +45,11 @@ def validate_asap_wrapper(request, *args, **kwargs):
 
 
 def requires_asap(issuers=None, subject_should_match_issuer=None, func=None):
+    warnings.warn(
+        "requires_asap in the contrib package is deprecated;"
```

Comment:
as `deprecated as of version 4.0.0` ?

Reply:
Technically, it's deprecated immediately as of the moment it's committed. :+1: 
If anything, we should be telling people when it will be discontinued completely.